react/modules/gaEvents.ts

```diff
@@ -118,3 +119,38 @@ export function selectPromotion(eventData: PixelMessage['data']) {
 
   updateEcommerce(eventName, data)
 }
+
+export function removeFromCart(eventData: PixelMessage['data']) {
+  if (!shouldMergeUAEvents()) return
+
+  const eventName = 'remove_from_cart'
+
+  const { items: eventDataItems, currency } = eventData
+
+  let totalValue = 0.0
+  const items = eventDataItems.map((item: CartItem) => {
+    const productName = getProductNameWithoutVariant(item.name, item.skuName)
+    const formattedPrice =
+      item.priceIsInt === true ? item.price / 100 : item.price
+
+    totalValue += formattedPrice
+
+    return {
+      item_id: item.productId,
+      item_brand: item.brand,
+      item_name: productName,
+      item_variant: item.skuId,
+      item_category: item.category,
+      quantity: item.quantity,
```

Comment:
Please check [this applicable comment](https://github.com/vtex-apps/google-tag-manager/pull/106#discussion_r1107715991).